### PR TITLE
reference to Teams in multi-geo scenario added

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/692

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/plan-for-multi-geo.md [hunk 1]
```diff
@@ -72,4 +72,6 @@ Read [User experience in a mult-geo environment](multi-geo-user-experience.md) f
 
 To get started configuring Office 365 Multi-Geo, see [Configure Office 365 Multi-Geo](multi-geo-tenant-configuration.md).
 
+For details about Teams experience in an Office 365 Multi-Geo tenancy, please see [Teams experience in an Office 365 OneDrive and SharePoint Online Multi-Geo-enabled tenancy](https://docs.microsoft.com/en-us/microsoftteams/teams-experience-o365odb-spo-multi-geo)
```
Comment (JohanFreelancer9) [inline Enterprise/plan-for-multi-geo.md]:
```suggestion
For details about the Teams experience in an Office 365 Multi-Geo tenancy, please see [Teams experience in an Office 365 OneDrive and SharePoint Online Multi-Geo-enabled tenancy](https://docs.microsoft.com/microsoftteams/teams-experience-o365odb-spo-multi-geo).
```

**Comment by mypil** [comment]
@VLG17 - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue #692

Thank you.

cc: @kenwith

**Pull request merged**